app/components/accepted-record/template.hbs

```diff
@@ -0,0 +1,30 @@
+<table class="table table-filter">
+    <tbody>
+        {{#each model.acceptedPreprints.records as |record|}}
+            <tr data-status={{record.reviewsState}}>
+                <td class="state">
+                    <a class="fa fa-check-circle-o fa-lg"> </a>
+                </td>
+                <td>
+                    <div class="media">
+                        <div class="media-body" title={{record.title}}>
+                            <a href="/preprints/{{theme.id}}/{{record.id}}">
+                                <p class="record-title">{{shorten-text record.title screenWidth}}</p>
+                                <p class="record-body">
+                                    submitted on {{moment-format record.dateCreated "MMMM DD, YYYY"}} by
```

Comment:
How the date is displayed on the preprints detail page:
`{{t logDateLabel}}: {{moment-format logDate "MMMM DD, YYYY"}} | {{t "content.header.last_edited"}}: {{moment-format model.dateModified "MMMM DD, YYYY"}}`